### PR TITLE
[FIX] Retry icon comes out of the div

### LINKED ISSUE #20389: Retry icon comes out of the box
### Steps to reproduce:

Any broken image in channel/room

### Expected behavior:

The retry icon should be inside the box

### Actual behavior:
Screenshot (41)_LI

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by im-adithya**
## Proposed changes (including videos or screenshots)

Changed the height of the div container.

## Issue(s)

This PR closes #20389
## Earlier

Screenshot (41)
## Now

Screenshot (44)

## Note

Due to different image sizes, the widths are different here in the image but actually the boxes are of the same width and the retry icon and text are of the same size.

### CODE REVIEW

**Comment by gabriellsh** [comment]
Instead of changing the sizes, wouldn't a simple padding in the placeholder suffice?

**Comment by gabriellsh** [comment]
Please, also post screenshots of the fix. Thanks!

**Comment by im-adithya** [comment]
> Instead of changing the sizes, wouldn't a simple padding in the placeholder suffice

Sure! This seems to be better indeed!!
I'll implement this

**Comment by im-adithya** [comment]
> Please, also post screenshots of the fix. Thanks!

Yeah I thought of posting but can you tell me how can I crash an image which is uploaded 😅

**Comment by im-adithya** [comment]
@gabriellsh I've made the necessary changes :)

**Comment by im-adithya** [comment]
@dougfabris @ggazzo can anyone please review this? Thanks!

File: client/components/Message/Attachments/components/Image.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const Retry: FC<BoxProps & { retry: () => void }> = ({ retry, ...props })
             background: var(--rxc-color-neutral-300, ${ colors.n300 }) !important;
         }
     `;
-	return <ImageBox className={clickable} {...props} onClick={retry}>
+	return <ImageBox className={clickable} {...props} onClick={retry} height='200px'>
```
Comment (tiagoevanp) [inline client/components/Message/Attachments/components/Image.tsx:35]: The design team of rocket.chat decided to use fixed sizes for Retry component, so I suggest to use the `size={160}` property in this component insted of `height='...'`.

**Review by tiagoevanp — CHANGES_REQUESTED** [review]
Other suggestion, remove the `{...props}` in that component, there is no necessity to pass dimensions for that.

File: client/components/Message/Attachments/components/Image.tsx (hunk 1 above)
Comment (im-adithya) [reply client/components/Message/Attachments/components/Image.tsx]: Done :)

**Review by tiagoevanp — DISMISSED** [review]
The `{ ...dimensions }` on Retry componente must be removed... anyways, approved.

**Comment by im-adithya** [comment]
Thank you @tiagoevanp! Removed {...dimensions} as well!

**Review by tiagoevanp — APPROVED** [review]

**Pull request merged**